.github/workflows/ci-test-playwright.yml

```
@@ -225,7 +226,7 @@ jobs:
       # continue-on-error: failed run keeps outcome=failure but conclusion=success — rerun must use outcome.
       - name: Run Playwright tests (attempt 1)
         id: attempt1
-        continue-on-error: true
+        continue-on-error: 
```

actionlint (1.7.12)

[error] expected bool value but found scalar node with "!!null" tag

(syntax-check)